### PR TITLE
Fix Eurostat EPO patents and trademark applications

### LINKED ISSUE #310: Investigate missing London region from Eurostat trademark indicator

### PR DESCRIPTION
**Pull request by Juan-Mateos**
Closes #310

### CODE REVIEW

File: ds/data/processed/eurostat/epo_patent_applications.nuts2.csv [hunk 1]
```diff
@@ -1,736 +1,841 @@
...
-1996,UKK3,2010,6
-1996,UKK4,2010,34
-1996,UKL1,2010,25
-1996,UKL2,2010,33
-1996,UKM2,2010,205
-1996,UKM3,2010,130
-1996,UKM5,2010,16
-1996,UKM6,2010,30
-1996,UKN0,2010,53
+2016,UKC1,2013,30
```
Comment (mindrones) [inline ds/data/processed/eurostat/epo_patent_applications.nuts2.csv]: Shouldn't the spec year here be 2016? (*)

File: ds/data/processed/eurostat/epo_patent_applications.nuts3.csv [hunk 2]
```diff
@@ -1,2626 +1,3634 @@
...
-1996,UKM63,2010,13.0
-1996,UKM64,2010,1.0
-1996,UKM65,2010,
-1996,UKM66,2010,1.0
-1996,UKN01,2010,12.0
-1996,UKN02,2010,12.0
-1996,UKN03,2010,14.0
-1996,UKN04,2010,
-1996,UKN05,2010,14.0
+2016,UKC11,2013,5.0
```
Comment (mindrones) [inline ds/data/processed/eurostat/epo_patent_applications.nuts3.csv]: Same as (*)

File: ds/data/processed/eurostat/eu_trademark_applications.nuts2.csv [hunk 3]
```diff
@@ -1,736 +1,841 @@
...
-1996,UKK3,2010,6
-1996,UKK4,2010,34
-1996,UKL1,2010,25
-1996,UKL2,2010,33
-1996,UKM2,2010,205
-1996,UKM3,2010,130
-1996,UKM5,2010,16
-1996,UKM6,2010,30
-1996,UKN0,2010,53
+2016,UKC1,2013,30
```
Comment (mindrones) [inline ds/data/processed/eurostat/eu_trademark_applications.nuts2.csv:2]: Same as (*)

File: ds/data/processed/eurostat/eu_trademark_applications.nuts3.csv [hunk 4]
```diff
@@ -1,2626 +1,3634 @@
...
-1996,UKM63,2010,13.0
-1996,UKM64,2010,1.0
-1996,UKM65,2010,
-1996,UKM66,2010,1.0
-1996,UKN01,2010,12.0
-1996,UKN02,2010,12.0
-1996,UKN03,2010,14.0
-1996,UKN04,2010,
-1996,UKN05,2010,14.0
+2016,UKC11,2013,5.0
```
Comment (mindrones) [inline ds/data/processed/eurostat/eu_trademark_applications.nuts3.csv:2]: Same as (*)

**Review by mindrones — CHANGES_REQUESTED** [review]
`UKI` is there correctly now but I guess NUTS spec years aren't correct (see below)?

File: ds/data/processed/eurostat/epo_patent_applications.nuts2.csv (hunk 1 above)
Comment (Juan-Mateos) [reply ds/data/processed/eurostat/epo_patent_applications.nuts2.csv]: No - this uses fixed NUTS categories. 2010 for patents and 2013 for trademarks.

Comment (mindrones) [reply ds/data/processed/eurostat/epo_patent_applications.nuts2.csv]: Ah right that's why we added the `warning`, never mind, sorry for the noise

**Review by mindrones — APPROVED** [review]

**Comment by mindrones** [comment]
On second thought, we need to delete `ds/data/processed/eurostat/epo_patent_applications.*.csv` as we discussed earlier.

**Comment by Juan-Mateos** [comment]
> On second thought, we need to delete `ds/data/processed/eurostat/epo_patent_applications.*.csv` as we discussed earlier.

Hey Luca, why do we need to delete those? I fixed the code so that patent applications and trademarks are now different.

File: ds/data/processed/eurostat/epo_patent_applications.nuts2.yaml [hunk 5]
```diff
@@ -28,7 +28,7 @@ schema:
 source_name: Eurostat
 source_url: https://ec.europa.eu/eurostat
 warning: The regional breakdown of the EU Member States is based on the Nomenclature of Territorial Units for Statistics – NUTS 2010 level 3
-year_range: [1996,2016]
+year_range: [1996,2012]
```
Comment (mindrones) [inline ds/data/processed/eurostat/epo_patent_applications.nuts2.yaml]: According to the data I think it should be

```suggestion
year_range: [2006,2012]
```

**Review by mindrones — APPROVED** [review]
Sorry I forgot to update the app data locally and it was showing me identical charts for the two indicators.

There is still an incorrect `year_range` to be fixed before merging, noted below.

**Review by mindrones — APPROVED** [review]
Good to merge! 👍

**Pull request merged**